hadoop-ozone/tools/src/main/java/org/apache/hadoop/ozone/freon/BaseFreonGenerator.java

```diff
@@ -548,6 +569,10 @@ public void setThreadNo(int threadNo) {
     this.threadNo = threadNo;
   }
 
+  /**
+   * Get current Thread sequence ID.
+   * @return Current Thread sequence ID
```

Comment:
Thanks for adding the comment, but I still don't think it's clear right away what it is used for.  The doc just repeats the method name.

Reply:
this has been updated